local_setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+curl -o tdr-realm-export.json https://github.com/nationalarchives/tdr-configurations/blob/master/keycloak/tdr-realm-export.json &&
+nvm install 16.17.0 &&
```

Comment:
This only needs to be done once to install npm so is it worth having it in the script? I tried this and I don't use nvm so this failed for me. I think most of us have npm installed now so this can probably come out.

Reply:
Cool. I'll add it to the README instructions instead, in case we have a new starter in the future.